doc/mgmt_preview_quickstart.md

```diff
@@ -0,0 +1,285 @@
+4.  Inside the System Properties window, click the Environment
+    Variables… button.
+5.  Click on the property you would like to change, then click the Edit…
+    button. If the property name is not listed, then click the New…
+    button.
+
+Linux-based OS :
+
+    export AZURE_CLIENT_ID="__CLIENT_ID__"
+    export AZURE_CLIENT_SECRET="__CLIENT_SECRET__"
+    export AZURE_TENANT_ID="__TENANT_ID__"
+    export AZURE_SUBSCRIPTION_ID="__SUBSCRIPTION_ID__"
+
+Authentication and Creating Resource Management Client
+------------------------------------------------------
+
+Now that the environment is setup, all you need to do is to create an
+authenticated client. Our default option is to use
+**DefaultAzureCredential** and in this guide we have picked
+**Resources** as our target service, but you can set it up similarly for
+any other service that you are using.
+
+To authenticate to Azure and create a management client, simply do the
+following:
+
+    using Azure.Identity;
+    using Azure.Management.Resource;
+    using Azure.Management.Resource.Models;
+    using System;
+    ...
+    var subscriptionId = Environment.GetEnvironmentVariable("AZURE_SUBSCRIPTION_ID");
+    var resourceClient = new ResourceClient(subscriptionId, new DefaultAzureCredential(true));
+
+More information and different authentication approaches using Azure
+Identity can be found in [this
+document](https://docs.microsoft.com/dotnet/api/overview/azure/identity-readme?view=azure-dotnet)
+
+Interacting with Azure Resources
+------------------
+
+Now that we are authenticated, we can use our management client to make API calls. Let's demonstrate management client's usage by showing concrete examples
+
+**Managing Resource Groups**
+
+We can use the Resource client (azure.mgmt.resource.ResourceManagementClient) we have created to perform operations on Resource Group. In this example, we will show to manage Resource Groups.
+
+***Create a resource group***
+
+    location = "myLocation";
+    groupName = "myResourceGroupName";
+    var result = await resourceClient.ResourceGroups.CreateOrUpdateAsync(groupName, new ResourceGroup(location));
+
+***Update a resource group***
+
+    ...
+    var tags = new Dictionary<string,string>();
+    tags.Add("environment","test");
+    tags.Add("department","tech");
+    resourceGroup.Tags = tags;
+
+    var updated = await resourceClient.ResourceGroups.CreateOrUpdateAsync(groupName, resourceGroup);
+
+***List all resource groups***
+    
+    AsyncPageable<ResourceGroup> response = resourceClient.ResourceGroups.ListAsync();
+    await foreach (ResourceGroup resourceGroup in response)
+    {
+        Console.WriteLine(resourceGroup.Name);
+    }
+
+***Delete a resource group***
+
+    await resourceClient.ResourceGroups.DeleteAsync(groupName);
+    
+Example: Creating a Virtual Machine
+-----------------------------------
+Let's show a concrete example of how you would create a virtual machine using .NET SDK
```

Comment:
Please refer to https://github.com/erich-wang/azure-sdk-for-net/blob/erich/mgmt.apireview/sdk/compute/Azure.Management.Compute/tests/CreateVMSample.cs for creating VM using track 2